packages/react/ds/src/header/components/header-search.tsx

```diff
@@ -14,28 +14,23 @@ function HeaderSearch({
 }) {
   const ActionType = action || serverAction;
   return (
-    <div
-      id="SearchContainer"
-      className={`gi-hidden gi-bg-gray-50 gi-px-8 gi-pt-8 gi-pb-14 gi-border-b-2xl gi-border-b-emerald-800`}
-    >
-      <form action={ActionType} className="gi-max-w-md gi-mx-auto">
-        <Heading as="h4">Search the website</Heading>
-        <div className="gi-flex gi-items-end gi-mt-4">
-          <TextInput
-            name="search_query"
-            aria-label="Search the website"
-            placeholder="Enter search term"
-            id="search"
-            type="text"
-            className="gi-flex-auto"
-          />
-          <Button className="gi-ml-1 gi-flex-none">
-            Search
-            <Icon icon={icon} ariaLabel="Search" />
-          </Button>
-        </div>
-      </form>
-    </div>
+    <form action={ActionType} className="gi-max-w-md gi-mx-auto">
```

Comment:
There is probably something wrong with the merge.
The current version of `main` has the correct styling.

Reply:
The styles are the same, I just moved the div container to the parent component, so in that case we show the right styles only on desktop.